components/menu/menu-item-styles.js

```diff
@@ -37,7 +37,13 @@ export const menuItemStyles = css`
 		z-index: 2;
 	}
 
-	:host([disabled]), :host([disabled]:hover), :host([disabled]:focus-visible) {
+	:host([disabled]), :host([disabled]:hover), :host([disabled].focus-visible) {
+		cursor: default;
+		opacity: 0.75;
+	}
+
+	/** separated because safari is having trouble parsing these */
+	:host([disabled]:focus-visible) {
```

Comment:
I'm confused as to why this needed the `:focus-visible` pseudo-class... shouldn't it have been handled by `.focus-visible` polyfill class in the ruleset above?

Reply:
You are right, it is redundant. I think I was just hesitant to remove it because I felt like the pseudo-class would be more stable through browser updates. I will remove them 👍

Reply:
Cool! Yeah I think once we have decent across-the-board browser support for native `:focus-visible`, we'll go through all these and switch them over. Need to wait probably another year and a half though for more people to have the latest Safari.